pygeos/test/test_misc.py

```diff
@@ -43,3 +46,65 @@ def foo():
 
     with pytest.raises(pygeos.UnsupportedGEOSOperation):
         foo()
+
+
+@multithreading_enabled
+def set_first_element(*args, **kwargs):
+    for arg in chain(args, kwargs.values()):
+        if hasattr(arg, "__setitem__"):
+            arg[0] = 42
```

Comment:
Suggestion: consider using a global constant, otherwise it gets a little harder to follow that 42 is being used as one (as in, it is an arbitrary rather than meaningful value).